x-pack/plugins/fleet/cypress/tasks/fleet.ts

```diff
@@ -41,7 +41,7 @@ export function navigateToAgentPolicy(name: string) {
 export function navigateToEnrollmentTokens() {
   cy.getBySel(ENROLLMENT_TOKENS_TAB).click();
   cy.get('.euiBasicTable-loading').should('not.exist');
-  cy.get('.euiButtonIcon--danger'); // wait for trash icon
+  cy.getBySel('enrollmentTokenTable.revokeBtn'); // wait for trash icon
```

Comment:
Thanks for tracking this! 
Could you move it to `x-pack/plugins/fleet/cypress/screens/fleet.ts` and assign it to a constant so it will be reusable elsewhere?

Reply:
👍 Done